tests/acceptance/test_onboarding.py

```diff
@@ -21,10 +24,10 @@ def setUp(self):
             teams=[self.team],
         )
         self.login_as(self.user)
-        self.path = '/onboarding/%s/' % self.org.slug
 
-    def test_onboarding(self):
-        self.browser.get(self.path)
+    @mock.patch('uuid.uuid4', return_value=UUID('031667ea1758441f92c7995a428d2d14'))
```

Comment:
@mattrobenolt I tried following the mocking of `uuid4`, but it's not '031667ea1758441f92c7995a428d2d14' as I would expect.

![image](https://user-images.githubusercontent.com/279398/31189794-175aa2a2-a8ee-11e7-82e0-8a3b8f6ff2b9.png)

I tried running...
```python
@mock.patch('uuid.uuid4', return_value=UUID('031667ea1758441f92c7995a428d2d14'))
def test(uuid4):
  print(uuid4().hex)

test()
```
...and that returns the expect value. Any ideas?

Reply:
Yeah, you need to patch the uuid that's imported into the package.

So your mock path needs to be something like `sentry.models.projectkey.uuid4`